internal/kubectl/kubectl_delete.go

```diff
@@ -4,8 +4,17 @@
 
 package kubectl
 
+import "github.com/elastic/elastic-package/internal/logger"
+
 // Delete function removes resources from the Kubernetes cluster based on provided definitions.
 func Delete(definitionsPath []string) error {
 	_, err := modifyKubernetesResources("delete", definitionsPath)
 	return err
 }
+
+// DeleteStdin function removes resources from the Kubernetes cluster based on provided definitions.
+func DeleteStdin(out []byte) error {
+	logger.Debugf("Delete Kubernetes stdin")
+	_, err := deleteKubernetesResourcesStdin(out)
+	return err
+}
```

Comment:
Previously, in Service Deployer, the elastic-agent in the kubernetes cluster was not deleted.
In the context of the PR / change, I think the elastic-agents should be deleted once the test is finished.

To be checked wether or not it would be possible to trigger as it is more than one test. Probably, it would be needed to create different names for the DaemonSet resource https://github.com/elastic/elastic-package/blob/a3e3a802b2d2c3f933c4e151f32cf12d5945005f/internal/servicedeployer/elastic-agent-managed.yaml.tmpl#L12